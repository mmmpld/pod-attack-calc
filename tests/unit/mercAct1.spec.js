import breakpoints from './breakpoints.js'
const characterId = 7;

describe('Merc Act1 default values', () => {
    it('standard attack', async () => {
        await breakpoints('14', '1.56', { charactersSelected: characterId });
    })
})