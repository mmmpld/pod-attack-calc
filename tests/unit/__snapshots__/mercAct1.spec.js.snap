// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Merc Act1 default values standard attack 1`] = `
<tbody>
  <tr class="highlight-current-row">
    <th>0</th>
    <td>14</td>
    <td>1.56</td>
  </tr>
  <tr class="">
    <th>9</th>
    <td>13</td>
    <td>1.66</td>
  </tr>
  <tr class="">
    <th>19</th>
    <td>12</td>
    <td>1.78</td>
  </tr>
  <tr class="">
    <th>32</th>
    <td>11</td>
    <td>1.92</td>
  </tr>
  <tr class="">
    <th>54</th>
    <td>10</td>
    <td>2.08</td>
  </tr>
  <tr class="">
    <th>86</th>
    <td>9</td>
    <td>2.27</td>
  </tr>
  <tr class="">
    <th>152</th>
    <td>8</td>
    <td>2.5</td>
  </tr>
</tbody>
`;
